src/main/java/models/Song.java

```diff
@@ -0,0 +1,143 @@
+package models;
+
+import enumeration.Genre;
+import exceptions.IncorrectSingerException;
+import exceptions.IncorrectSongTitleException;
+import exceptions.InvalidSongTimingException;
+
+/**
+ * This class represent song, which will be executed by the audio player
+ */
+public class Song {
+    private String title;
+    private Genre genre;
+    private Singer singer;
+    private int timing;
+
+    /**
+     * The constructor initialize the class fields with the values from the parameters if they are valid, otherwise it
+     * initialize the fields with default values
+     *
+     * @param title  - this parameter is the title of the song
+     * @param singer - this parameter is the singer of the song
+     * @param genre  - this parameter is the genre of the song
+     * @param timing - this parameter is timing of the song
+     */
+    public Song(String title, Singer singer, Genre genre, int timing) {
+        if (!isTitleValid(title)) {
```

Comment:
maybe call the method is{something}Invalid and return the needed result to avoid '!' and maybe have all the validations in one method.